[2021-08-10 02:32:00,913] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2021-08-10T02:31:59.977766+00:00 [queued]>
[2021-08-10 02:32:00,928] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2021-08-10T02:31:59.977766+00:00 [queued]>
[2021-08-10 02:32:00,929] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-08-10 02:32:00,929] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-08-10 02:32:00,929] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-08-10 02:32:00,936] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_1> on 2021-08-10T02:31:59.977766+00:00
[2021-08-10 02:32:00,938] {standard_task_runner.py:52} INFO - Started process 222527 to run task
[2021-08-10 02:32:00,943] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_1', '2021-08-10T02:31:59.977766+00:00', '--job-id', '28', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp342w_i0e', '--error-file', '/tmp/tmp3bzdec6w']
[2021-08-10 02:32:00,944] {standard_task_runner.py:77} INFO - Job 28: Subtask task_1
[2021-08-10 02:32:01,044] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_1 2021-08-10T02:31:59.977766+00:00 [running]> on host airflowvm
[2021-08-10 02:32:01,161] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_1
AIRFLOW_CTX_EXECUTION_DATE=2021-08-10T02:31:59.977766+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-10T02:31:59.977766+00:00
[2021-08-10 02:32:01,162] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-08-10 02:32:01,162] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-08-10 02:32:01,170] {subprocess.py:75} INFO - Output:
[2021-08-10 02:32:04,172] {subprocess.py:83} INFO - Command exited with return code 0
[2021-08-10 02:32:04,203] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_1, execution_date=20210810T023159, start_date=20210810T023200, end_date=20210810T023204
[2021-08-10 02:32:04,235] {taskinstance.py:1245} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2021-08-10 02:32:04,253] {local_task_job.py:151} INFO - Task exited with return code 0
